index.html

```diff
@@ -247,6 +176,42 @@ <h2>About me</h2>
+                                <div class="child-class">
+                                    <div class="two-columns">
+                                        <h4 class="heading-child">Keeping track of hundreds of
+                                            components</h4>
+                                        <div class="display-centered">
+                                            <a href="" class="btn-secondary">See Live <img src="images/icons/live.svg" alt="See PRoject Live"/></a>
+                                            <a href="" class="btn-secondary">See Source <img src="images/icons/github.svg" alt="See Source Code"/></a>
+                                        </div>
+                                    </div>
+                                    <div class="childs">
+                                        <div class="asscories">
+                                            <ul class="card-tags">
+                                                <li>Codekit</li>
+                                                <li>Github</li>
+                                                <li>Javascript</li>
+                                                <li>Bootstrap</li>
+                                                <li>Terminal</li>
+                                                <li>Codepen</li>
+                                            </ul>
+                                        </div>
+                                        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. em Ipsum is simply dummy text of the printing and typesetting industry. em Ipsum is simply dummy text of the printing and typesetting industry. em Ipsum is simply dummy text of the printing and typesetting industry. em Ipsum is simply dummy text of the printing and typesetting industry. em Ipsum is simply dummy text of the printing and typesetting industry. em Ipsum is simply dummy text of the printing and typesetting industry. em Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+                                    </div>
+                                </div>
+                
+                            </div>
+                        </div>
+                    </section>
```

Comment:
- [ ] Kindly note that the popup is supposed to be created dynamically and not to be hardcoded as it currently is right now. To achieve this you will need the following implemented:

1.   An array which has objects with the different fields for the popup i.e have something that looks like this

```
let modalpopup = [
{
    id: '1',
    title: 'Multi Post Stories',
    closebtn: '<i class="fa fa-times" aria-hidden="true"></i>',
    html: 'html',
    bootstrap: 'bootstrap',
    ruby: 'Ruby On Rails',
    projectImage: './images/mobilemodal.png',
    projectDesc: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam amet animi voluptate doloribus at quaerat, vitae dicta sunt possimus voluptatum, facilis vel? Eligendi, harum ipsum excepturi sint at quasi? Repellendus eveniet suscipit quibusdam inventore architecto minus eligendi adipisci. Harum atque voluptate dolore fugiat pariatur eius ullam architecto porro aspernatur nihil.',
    projectbtnlive: 'https://nemwel-boniface.github.io/personal-portfolio/',
    projectbtnsource: 'https://nemwel-boniface.github.io/personal-portfolio/',
  },
]
````
2. When the projects are being created and rendered to the screen, attach event listeners to them so that when someone click on them, the display popup method is called and the popup is displayed. You could add corresponding id's to the buttons when created so that they correspond the the id of the popup array and when you click on the show project button, the display popup only shows the popup with the id same to the one of the button that was clicked.
3. Implementing the change above will allow you to render different popups for each project that you want to show in your screen.

**This is almost all that you need to implement it dynamically as in the project requirements. Exactly as how the projects section is done**


- [ ] Kindly as requested of you by the previous reviewer, kindly ensure that the vertical scroll is not there in your design so that your design matches the Figma design for both mobile and desktop view

[Screencast from 07-02-2023 06:12:37 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/217284167-1a4b3126-4e90-4c9b-90f9-99c5429f18ef.webm)
.

Reply:
@Nemwel-Boniface   please understand that I have followed as much as possible the figma template and the popup cannot appear completely in one click , there are more contents that need to be seen using a scroll . 
@Nemwel-Boniface  I also think it creates a  better user experience when it is available coz without it the popup would only be able to display half of its contents